test/Microsoft.AspNet.Mvc.Xml.Test/FlushReportingStream.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Threading;
+using Moq;
+
+namespace Microsoft.AspNet.Mvc.Core
```

Comment:
Commit 670ca203: Fixed the namespace in Microsoft.AspNet.Mvc.Xml.Test to be Microsoft.AspNet.Mvc.

Let me know if you want me to fix them in Microsoft.AspNet.Mvc.Core.Test?